src/utils/round.ts

```diff
@@ -0,0 +1,2 @@
+export const roundToHundret = (number: number): number =>
```

Comment:
[CODE STYLE] - The function name 'roundToHundret' contains a spelling mistake. It should be 'roundToHundred'. Additionally, the function actually rounds to two decimal places, so a more appropriate name might be 'roundToTwoDecimals' to avoid any confusion.